packages/opentelemetry-core/src/context/context.ts

```diff
@@ -84,3 +91,28 @@ export function getParentSpanContext(
 ): SpanContext | undefined {
   return getActiveSpan(context)?.context() || getExtractedSpanContext(context);
 }
+
+/**
+ * Set whether or not instrumentation should be suppressed beyond
```

Comment:
Nit: "Set <what> wether ...."

Reply:
Whether is correct in this case. Wether is a castrated ram, weather is rain, whether expresses a choice.

Reply:
lol, i'm glad you double-checked. i was just like "yeah, that's a word i'd misspell."

Reply:
Sry it was leftover I made a typo but it wasn't what I really meant :/, the description and a function is a bit difficult to understand what it really does but it looks like I haven't finished this part when reviewing and switching between tabs. Anyway, the function itself `suppressInstrumentation` means "it will suppress instrumentation". Also having an extra optional param which by default is true is opposite to what usually the extra optional param value is. Function name suggests pretty straightforward what it will do so having an extra param to make opposite action of suppress is a bit misleading. I would either change a name to something like "setSuppressValue" or create a new function which will do opposite action to suppress, finding appropriate name I would leave to you then, but maybe "permitInstrumentation", "allowInstrumentation", etc.. So either something like `setSuppressValue` or add extra function which will do opposite action to suppress, WDYT ?

Reply:
I think adding another function is the most straightforward. `suppressInstrumentation` and `unsuppressInstrumentation`

Reply:
I think that is a valid point. I'd originally started out with the "setX" approach but there was a point made that for the standard (90%?) case, a user will just be suppressing and the suppressInstrumentation w/o needing to pass the bool would be a bit nicer.

Naming is always fun. `unsuppressInstrumentation` is probably fine if we add another function, but feels awkward. I suppose the important part is it is understood as the pair to `suppressInstrumentation` where other combinations of words may not be so obvious.